### ISSUE TITLE
Indent rule creates compile error on multi-line string arguments

### ISSUE THREAD

**Issue by Deco354** [nicklockwood/SwiftFormat#562]
I get a compile error `Insufficient indentation of next 4 lines in multi-line string literal` when applying the `indent` rule on the following code. (
```swift
        XCTAssertEqual(sessionTimeoutEvent.script?.source, """
        window.addEventListener('message', event => {
        if (event.data && event.data.type && event.data.type === 'GL.TIMEOUT')
        window.webkit.messageHandlers.SportsNativeSessionTimeout.postMessage('');
        });
            """)
```

I can resolve this by extracting the multi-line string to a variable as seen below but figured I'd report it all the same to avoid this happening to others in future.

### Fix
```swift
        let expectedTimeoutEvent = """
        window.addEventListener('message', event => {
        if (event.data && event.data.type && event.data.type === 'GL.TIMEOUT')
        window.webkit.messageHandlers.SportsNativeSessionTimeout.postMessage('');
        });
        """
        XCTAssertEqual(sessionTimeoutEvent.script?.source, expectedTimeoutEvent)
```

### Versions
swift: 5.1
swiftformat: 0.43.4
Xcode: 11.3
OSX: 10.14.6

### Screenshot
![image](https://user-images.githubusercontent.com/18557720/74224702-a0e14500-4cb9-11ea-8715-9f0d3b1d8577.png)

### Indent settings
```
--indent 4
--xcodeindentation "enabled"
--ifdef "no-indent"
```

**Comment by nicklockwood** [comment]
@Deco354 sorry for the delay, I'll get it fixed asap.

**Comment by PompeiaPaetenari** [comment]
I also encountered this for any time I pass a multiline string as a function parameter by switching between using tabs and using spaces.  Everything seems to remain the same except the closing quotes, which end up indented differently.

**Comment by nicklockwood** [comment]
@Deco354 @PompeiaPaetenari fixed in 0.44.3.
